### DISCUSSION TITLE
Need Help

### DISCUSSION THREAD

**Discussion by Doubletrax1979** [MarcDonald/twitch-spotify-request-bot#31] [Q&A]
When it come to this, i will say that im a noob. Having several issues trying to set this up. Anyone available to help with step by step instructions.  I have went to two different people and ive had no luck

**Comment by MarcDonald** [comment]
Hey @Doubletrax1979, I updated some of the instructions and fixed some issues in [version 5.0.0](https://github.com/MarcDonald/twitch-spotify-request-bot/releases/tag/v5.0.0). Let me know if this helps at all and/or what steps aren't working as intended
